packages/block-library/src/navigation/edit/index.js

```diff
@@ -342,16 +342,7 @@ function Navigation( {
 	const [ detectedOverlayColor, setDetectedOverlayColor ] = useState();
 
 	const onSelectClassicMenu = async ( classicMenu ) => {
-		const navMenu = await convertClassicMenu(
-			classicMenu.id,
-			classicMenu.name,
-			'draft'
-		);
-		if ( navMenu ) {
-			handleUpdateMenu( navMenu.id, {
-				focusNavigationBlock: true,
-			} );
-		}
+		await convertClassicMenu( classicMenu.id, classicMenu.name, 'draft' );
```

Comment:
```suggestion
		return convertClassicMenu( classicMenu.id, classicMenu.name, 'draft' );
```

No need to return await inside an `async` function as it will _always_ return a `Promise`.